specs/_features/whisk/beacon-chain.md

```diff
@@ -382,49 +367,30 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 
 ### Deposits
 
+```python
+def get_initial_whisk_k(validator_index: ValidatorIndex, counter: int) -> BLSFieldElement:
+    return BLSFieldElement(bytes_to_bls_field(hash(uint_to_bytes(validator_index) + uint_to_bytes(uint64(counter)))))
+```
+
 ```python
 def get_unique_whisk_k(state: BeaconState, validator_index: ValidatorIndex) -> BLSFieldElement:
     counter = 0
     while True:
         # hash `validator_index || counter`
-        k = BLSFieldElement(bytes_to_bls_field(hash(uint_to_bytes(validator_index) + uint_to_bytes(uint64(counter)))))
+        k = get_initial_whisk_k(validator_index, counter)
         if is_k_commitment_unique(state, BLSG1ScalarMultiply(k, BLS_G1_GENERATOR)):
             return k  # unique by trial and error
         counter += 1
 ```
 
 ```python
-def get_initial_commitments(k: BLSFieldElement) -> Tuple[BLSG1Point, WhiskTracker]:
-    return (
-        BLSG1ScalarMultiply(k, BLS_G1_GENERATOR),
-        WhiskTracker(r_G=BLS_G1_GENERATOR, k_r_G=BLSG1ScalarMultiply(k, BLS_G1_GENERATOR))
-    )
+def get_k_commitment(k: BLSFieldElement) -> BLSG1Point:
```

Comment:
Nitpick: Maybe rename to `compute_k_commitment()`? This gives the impression that it's a getter method, but it's actually computing something from scratch.


(Feel free to ignore this, since we are likely not consistent about this anyway)